docs/apps/srt-live-transmit.md

```diff
@@ -183,6 +185,19 @@ instead of `IP_ADD_MEMBERSHIP` and the value is set to `imr_sourceaddr` field.
 Explanations for the symbols and terms used above can be found in POSIX
 manual pages, like `ip(7)` and on Microsoft docs pages under `IPPROTO_IP`.
 
+### Medium: RTP
+
+RTP over UDP is supported for input only.
+
+With no RTP-specific URI parameters, RTP medium functions identically to UDP medium as described above.
+
+Additional RTP-specific options are available through URI parameters. These options enable dropping of bytes from the head of the input buffer allowing, for example, the RTP header to be dropped for use cases where the receiving end cannot accept RTP.
+
+- **droprtpheader**: (`bool` as defined in SRT section) - when true, drop the first 12 bytes of each received packet
+- **rtpheadersize**: sets the number of bytes to drop from the beginning of each received packet. Ignored if **droprtpheader** is not truthy.
```

Comment:
Not sure what was the intention.

RTP always DOES have a header. What the application is actually reading on the input is the PAYLOAD. RTP is only a way to wrap the payload. This means that with RTP input you read a wrapped format into RTP, so the header should be always dropped. Normally it should also verify if the RTP header "makes sense" and also parse its content, if needed to recognize its actual length, but it doesn't have to be that complicated - it suffices if it simply relies on the user's declaration about the length. But then the default 12 bytes should be treated as a length, with a possibility for the user to specify a different size (and I guess, 12 should be also the minimum allowed value).

Reply:
I had assumed that some users may wish to continue to pass the RTP headers downstream where the receiving client can handle them but also set the scheme to `rtp`. Hence the defaulting to `false` for dropping the header. I understand that those users could just use `udp` but that feels unexpected if the option to set rtp is available.

Please could you clarify the change(s) that you would like to see to resolve this?

Thanks for the review so far.

Reply:
Clarity was provided in [the previous comment](https://github.com/Haivision/srt/pull/2848#discussion_r1447162048), and the latest commit ([03e6cdf](https://github.com/Haivision/srt/pull/2848/commits/03e6cdfc2153c0c8c2bd336e44e68c6650399a96)) should hopefully give what's needed.